packages/monitoring-system/README.md

```diff
@@ -1,3 +1,82 @@
-#  GitHub Automation Bots Monitoring System 
+#  GitHub Automation Bots Monitoring System (Internal Only)
 
-// TODO(asonawalla)
\ No newline at end of file
+A system to collect metrics on GitHub Repo Automation Bot executions and make it easily accessible to bot SREs and developers for debugging and facilitating the development process.
+
+[Read the design document here (internal)](http://go/automation-bot-monitoring-system-design)
+
+[Watch the presentation (starting 17m15s)](https://drive.google.com/file/d/1lmMbKiRUkE1BaCo4bR9lWKXJxOdaR01P/view&sa=D&ust=1598459279983000&usg=AFQjCNEA1FGlFyM4XtH2KaMgnvDjiLZT1g)
```

Comment:
Let's not link internal accessible links here

Reply:
Is that for just this link or should I remove all internal links?

Reply:
Probably all of them.